src/libc.cr

```diff
@@ -1,29 +1,32 @@
 lib LibC
+  ifdef x86_64
+    alias SizeT = UInt64
+    alias SSizeT = Int64
+  else
+    alias SizeT = UInt32
+    alias SSizeT = Int32
+  end
+
+  #ifdef windows
```

Comment:
Is this code supposed to be commented out? If so, I'd say it'd be better to just remove it.